libraries/AP_AccelCal/AP_AccelCal.cpp

```diff
@@ -168,10 +177,18 @@ void AP_AccelCal::update()
             _last_position_request_ms = now;
             switch (_last_result) {
                 case ACCEL_CAL_SUCCESS:
+#if !defined(HAL_BUILD_AP_PERIPH)
```

Comment:
breaking out a `send_vehicle_position` would reduce the number of places we need to `#if` - readability win

Reply:
It's only three places, also the behaviour is similar in only these two cases, for the third one it's supposed to just ignore it, don't think its worth it to create another thing to scroll to 😄 .